### PR TITLE
feat: add Oracle Docker images to the worker cache

### PR DESCRIPTION
**Pull request by kuisathaverat**
* Add Oracle Docker images to the packer images of immutable workers

related to https://github.com/elastic/beats/pull/12890

### CODE REVIEW

**Comment by kuisathaverat** [comment]
failed test sare no related to the PR, it only contains a new file used by our infra scripts, not the beats-ci

**Comment by kuisathaverat** [comment]
@jsoriano @sayden Could we merge it? it is a blocker for #12890

File: .ci/packer_cache.sh [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+# python:3.6-alpine
+# quay.io/coreos/etcd:v3.3.10
+# rabbitmq:3.7.4-management
+# redis:3.2.12-alpine
+# redis:3.2.4-alpine
+# store/oracle/database-enterprise:12.2.0.1
+# traefik:1.6-alpine
+# tsouza/nginx-php-fpm:php-7.1
+# ubuntu:16.04
+# ubuntu:trusty
```
Comment (jsoriano) [inline .ci/packer_cache.sh]: Not sure if we discussed this in the past, but how is this list of images generated?

Comment (kuisathaverat) [reply .ci/packer_cache.sh]: low technology bash scripting :)

`find . -name Dockerfile -exec grep -- FROM {} + | cut -d " " -f 2`

**Comment by kuisathaverat** [comment]
@jsoriano If you are ok with this change we can merge it to add the Docker image to the cache, it takes a couple of days or so, then we can test the other PR.

**Pull request merged**